torch/csrc/generic/StorageMethods.cpp

```diff
@@ -96,7 +96,7 @@ static PyObject * THPStorage_(fromBuffer)(PyObject *_unused, PyObject *args, PyO
   PyObject *obj = nullptr;
   const char* byte_order_str = nullptr;
   Py_ssize_t count = -1, offset = 0;
-  Py_buffer buffer;
+  Py_buffer buffer = {};
```

Comment:
Because i've touched this file, i was getting a clang-tidy error on this line. This is my attempt to fix it but i need fully green build to verify that it works.